src/Workspaces/MSBuild/Core/Rpc/RemoteBuildHost.cs

```diff
@@ -23,22 +23,22 @@ public RemoteBuildHost(RpcClient client)
     }
 
     public Task<bool> HasUsableMSBuildAsync(string projectOrSolutionFilePath, CancellationToken cancellationToken)
-        => _client.InvokeAsync<bool>(BuildHostTargetObject, nameof(BuildHost.HasUsableMSBuild), parameters: [projectOrSolutionFilePath], cancellationToken);
+        => _client.InvokeAsync<bool>(BuildHostTargetObject, "HasUsableMSBuild", parameters: [projectOrSolutionFilePath], cancellationToken);
```

Comment:
Would it be crazy to add all these methods to a shared interface so that you could continue to use `nameof()` and ensure the names don't change from beneath you?